ERROR: Failed to build gem native extension

```
$ ruby -v
ruby 2.7.0dev (2019-02-08 trunk 67037) [x86_64-linux]

$ gem install racc
Building native extensions. This could take a while...
ERROR:  Error installing racc:
        ERROR: Failed to build gem native extension.

    current directory: /home/u/.rbenv/versions/2.7.0-dev/lib/ruby/gems/2.7.0/gems/racc-1.4.14/ext/racc
/home/u/.rbenv/versions/2.7.0-dev/bin/ruby -I /home/u/.rbenv/versions/2.7.0-dev/lib/ruby/2.7.0 -r ./siteconf20190208-29225-1fcnlq9.rb extconf.rb
checking for rb_ary_subseq()... yes
creating Makefile

current directory: /home/u/.rbenv/versions/2.7.0-dev/lib/ruby/gems/2.7.0/gems/racc-1.4.14/ext/racc
make "DESTDIR=" clean

current directory: /home/u/.rbenv/versions/2.7.0-dev/lib/ruby/gems/2.7.0/gems/racc-1.4.14/ext/racc
make "DESTDIR="
make: *** No rule to make target '/home/u/.rbenv/versions/2.7.0-dev/include/ruby-2.7.0/defines.h', needed by 'cparse.o'.  Stop.

make failed, exit code 2

Gem files will remain installed in /home/u/.rbenv/versions/2.7.0-dev/lib/ruby/gems/2.7.0/gems/racc-1.4.14 for inspection.
Results logged to /home/u/.rbenv/versions/2.7.0-dev/lib/ruby/gems/2.7.0/extensions/x86_64-linux/2.7.0-static/racc-1.4.14/gem_make.out
```

Related to [Ruby@67036](https://github.com/ruby/ruby/commit/74f94b3e6ebf15b76f3b357e754095412b006e94)?

--- a/ext/racc/depend
+++ b/ext/racc/depend
@@ -1,1 +1,1 @@
-cparse.o: cparse.c $(hdrdir)/ruby.h $(topdir)/config.h $(hdrdir)/defines.h
+cparse.o: cparse.c $(hdrdir)/ruby/ruby.h $(topdir)/config.h $(hdrdir)/ruby/defines.h

